keyserver/internal/internal.go

```diff
@@ -513,6 +513,18 @@ func (a *KeyInternalAPI) uploadLocalDeviceKeys(ctx context.Context, req *api.Per
 }
 
 func (a *KeyInternalAPI) uploadOneTimeKeys(ctx context.Context, req *api.PerformUploadKeysRequest, res *api.PerformUploadKeysResponse) {
+	if len(req.OneTimeKeys) == 0 {
```

Comment:
Can we add an assertion to fail the request if `req.UserID` or `req.DeviceID` are empty?